core/vm/actor/builtin/v5/market/validate.hpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "vm/actor/builtin/states/market/market_actor_state.hpp"
+#include "vm/actor/builtin/types/market/policy.hpp"
+#include "vm/exit_code/exit_code.hpp"
+
+namespace fc::vm::actor::builtin::v5::market {
+  using primitives::DealId;
+  using primitives::SpaceTime;
+  using primitives::address::Address;
+  using primitives::piece::PaddedPieceSize;
+
+  struct validate_RESULT {
```

Comment:
```suggestion
  struct ValidateResult {
```

Reply:
i don't like `ValidateResult` because it changes `validate` and doesn't split it from "result".
this is like named tuple, used only by this function.
may rename if you want and will suggest name (please confirm to rename to `ValidateResult`).

Reply:
I think that we should to rename it, because it is a struct after all.